parquet2hive/parquet2hivelib.py

```diff
@@ -27,7 +27,8 @@ def get_bash_cmd(dataset, success_only = False, recent_versions = None, version
     bash_cmd, versions_loaded = "", 0
     for (version_prefix, dataset_name, version) in versions:
         sample, success_exists = "", False
-        for key in bucket.objects.filter(Prefix=version_prefix):
+        keys = sorted(bucket.objects.filter(Prefix=version_prefix), key = lambda obj : obj.last_modified, reverse = True)
```

Comment:
So it takes on the order of 20s to fetch all the main_summary objects (for v3). This is 7373 objects. Do we need to optimize? Or merge this PR now and optimize later?